### DISCUSSION TITLE
a basic plugin tutrial?

### DISCUSSION THREAD

**Discussion by masiha128** [Impostor/Impostor#334] [Q&A]
please help :)

**Comment by JohnTranQUT** [comment]
> please help :)

I've found the documentation is sufficient and example plugins to be very useful.

Examples can be found in the development discord. They are pinned there.

Opening up the impostor dll library has revealed a lot of event listeners. In Visual Studios 2019, the library are well linked so intellisense (class and code suggestion) was easy to follow.
